functions/src/api/authentificateJWT.spec.ts

```diff
@@ -0,0 +1,5 @@
+describe('authentificateJWT', () => {
```

Comment:
spelling errror: authentificate -> authenticate, as it should follow the file name `authenticateJWT`